extensions/wikia/Recirculation/WikiRecommendations.class.php

```diff
@@ -591,20 +609,21 @@ class WikiRecommendations {
 			'url' => 'http://midnight-texas.wikia.com/wiki/Midnight,_Texas_Wikia',
 		]
 	];
-	
-	public static function getRecommendations( $contentLanguage ) {
-		global $wgDevelEnvironment;
-
 
-		if ( empty( $wgDevelEnvironment ) ) {
-			$recommendations = self::RECOMMENDATIONS['en'];
+	public static function getRecommendations( $contentLanguage ) {
+		global $wgWikiaEnvironment;
 
-			if ( array_key_exists( $contentLanguage, self::RECOMMENDATIONS ) ) {
-				$recommendations = self::RECOMMENDATIONS[$contentLanguage];
-			}
-			shuffle( $recommendations );
+        if ( $wgWikiaEnvironment == WIKIA_ENV_STAGING ){
```

Comment:
Both should use strict equals `===` per conventions.

Reply:
Done